tests/aggregation_regress/tests.py

```diff
@@ -430,13 +430,17 @@ def test_decimal_aggregate_annotation_filter(self):
 
     def test_field_error(self):
         # Bad field requests in aggregates are caught and reported
-        with self.assertRaises(FieldError):
+        msg = "Cannot resolve keyword 'foo' into field. Choices are: authors, contact, contact_id, hardbackbook, " \
```

Comment:
Is 79 a hard limit? There are a few places where a single name could be fitted onto the previous line which would exceed the limit (but only by a little bit).